### DISCUSSION TITLE
Nothing added to my realtime logs when endpoint is hit

### DISCUSSION THREAD

**Discussion by Andi365** [vercel/community#1610] [Help] [answered]
### Summary

Hi, I dont understand why nothing is showing up on my realtime logs. Its really hard to debug if everything works this way. At first I even thought that nothing worked, but it seems to be making the changes it needs to.

I know the functions run and the "waiting for connection" spinner goes away when I make the request, but nothing shows up on the screen.

The app is made with Remix and its just an api route that has a loader with all the logic in it. 

Has anyone tried this before?

**Comment by TooTallNate** [comment]
Are you still experiencing this issue? I tried it just now and it worked as expected:

<img width="262" alt="Screenshot 2023-02-28 at 12 58 10 AM" src="https://user-images.githubusercontent.com/71256/221803399-70437a49-8b6d-46df-a4f8-a9a975836227.png">

<img width="168" alt="Screenshot 2023-02-28 at 12 57 46 AM" src="https://user-images.githubusercontent.com/71256/221803412-98ee909a-a619-4556-bba7-8abb9964dba5.png">

**Comment by TooTallNate** [comment, marked as answer]
We've identified and fixed an issue related to serverless function logs reporting, so you should be able to properly debug failing routes once again. Please let us know if you still are having any issues related to that!
